gradle/Dockerfile

```diff
@@ -1,7 +1,43 @@
 FROM ghcr.io/dependabot/dependabot-updater-core
 
+# Install Java
+RUN apt-get update && apt-get install -y --no-install-recommends \
+    openjdk-11-jdk \
```

Comment:
Any particular reason why you chose this version? It's quite old, and the latest LTS is 21.

Reply:
I used https://github.com/dependabot/dependabot-core/compare/main...jakecoffman/maven-with-mvn as a starting point for this section, guess it was a bit outdated. I can switch to 21 and see if that breaks anything.